openshift-kube-apiserver/admission/autoscaling/managementcpusoverride/admission.go

```diff
@@ -239,6 +239,13 @@ func (a *managementCPUsOverride) Admit(ctx context.Context, attr admission.Attri
 		return nil
 	}
 
+	// we should skip mutation of the pod that has container with both CPU limit and request because once we will remove
```

Comment:
Why do we want to modify nothing instead of modifying both?

Reply:
So a pod modification 1) can't result in a pod QoS class change and 2) can't happen on guaranteed pods with a container where only `limits` is specified (in which case `requests` will default to `limits`) or both `limits` and `requests` are specified and are equal.  Is this correct?

Worth noting, we should have no guaranteed pods in OCP.

Did we actually encounter this edge case? If so, in which component?

Reply:
I got it under the ACM pod. I will drop you the link under the slack.

@deads2k To modify both we will need to have an additional set of PRs for the kubelet and CRI-O because once we replace the limit the runtime needs to set the container CPU quota correctly.
It a big change for 4.8(code freeze is coming), but we have plans to do it for 4.9.